docker/ceph/Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN curl -LsS https://raw.githubusercontent.com/ceph/ceph/"$VCS_BRANCH"/ceph.spe
     -o /ceph/ceph.spec.in
 
 ARG FOR_MAKE_CHECK=1
+ARG _SOURCED_LIB_BUILD=1
```

Comment:
yeah it feels hacky and it won't import that library as well. I am not sure if we need that library in ceph-dev build since its something related to jenkins build only.